tests/crypto_aesctr/main.c

```diff
@@ -410,6 +474,17 @@ selftest(void)
 	if (selftest_cases(tests_256, num_tests, 0))
 		failures++;
 
+	/* Test with nonce=*2 (decimal). */
```

Comment:
I'm confused by "nonce=*2 (decimal)" here.  It looks like you've set a nonce of "2222222" in base 1024?

Reply:
I mean, there isn't anything especially wrong with this value, but it's a strange choice.  I would have gone with something 0x1234567890ABCDEF.

Reply:
I wanted something higher than `0x8` to check that it wasn't `int64_t` or something like that.  How about `0xfedcba987654321`?

Reply:
I think you mean 0xfedcba9876543210, not 0xfedcba987654321, but sure.

I think the nonce value you were calculating was 0x4010040100401004, so not high enough to run into signed issues anyway...